actors/builtin/miner/deadlines_test.go

```diff
@@ -121,15 +121,17 @@ func TestProvingPeriodDeadlines(t *testing.T) {
 			assert.False(t, di0.HasElapsed())
 			assert.True(t, di0.FaultCutoffPassed())
 
-			// The next not-elapsed is this one, which is still open
+			// The next not-elapsed is this one, which is not yet
+			// open, but not elapsed either.
 			nxt0 := di0.NextNotElapsed()
 			assert.Equal(t, periodStart, nxt0.PeriodStart)
 			assert.Equal(t, uint64(0), nxt0.Index)
 
 			di1 := miner.NewDeadlineInfo(periodStart, 1, curr)
 			assert.False(t, di1.IsOpen())
 			assert.False(t, di1.HasElapsed())
-			assert.False(t, di1.FaultCutoffPassed())
+			// The fault cutoff is more than one deadline into the future.
+			assert.True(t, di1.FaultCutoffPassed())
```

Comment:
@anorth is this going to be a problem? (this is due to the timing changes).

Reply:
Slightly annoying, but not a significant problem.